### ISSUE TITLE
View Transitions Module Level 2

### ISSUE THREAD

**Issue by xiBread** [Fyrd/caniuse#7028]
The draft for [View Transitions Module Level 2](https://drafts.csswg.org/css-view-transitions-2/) was just made available a week ago, adding support for cross-document navigation.

**Comment by bramus** [comment]
This has been available in Chrome for quite some time and is, as of recently, also available in Safari Technology Preview.

Authors are actively asking about support, so would be good to add it. It’s effectively an alias for the https://caniuse.com/mdn-css_at-rules_view-transition entry.

**Comment by davidwebca** [comment]
Yes. Safari just announced the release in 18.2 so it would be important to add this one soon.

**Comment by davidwebca** [comment]
@Fyrd Hi. Sorry for pinging. Is there anything I can do to help to have this one bumped up?

**Comment by Fyrd** [comment]
Now available at https://caniuse.com/cross-document-view-transitions
